akka-docs/rst/java/remoting.rst

```diff
@@ -506,13 +481,83 @@ as well as the `Oracle documentation on creating KeyStore and TrustStores <https
 are both great resources to research when setting up security on the JVM. Please consult those resources when troubleshooting
 and configuring SSL.
 
+Since Akka 2.5.0 mutual authentication between TLS peers is enabled by default. In previous releases this option was 
+available, recommended however not enabled by default. If you're rolling out a cluster and migrating it from Akka 2.4 to 2.5 
+make sure to enable this option on the 2.4 nodes during the rollout, otherwise it will not be Enable mutual authentication with configuration property::
```

Comment:
Not sure we should complicate the documentation with such stuff, since we have the full instructions written in the migration guide.